microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Prerelease testing
+description: Prerelease testing.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/09/2024
+---
+# Prerelease testing
+
+Prerelease testing helps apps catch bugs in prerelease channels of WebView2, before they ship in a stable release and degrade the app experience for end users. This document provide guiding principles and tools to help partners get started.
+
+<!-- ====================================================================== -->
+#### What is prerelease testing?
+Prerelease testing means validating your app against early builds of WebView2 to ensure that your app works with upcoming changes. This can be accomplished through automated testing and flighting with Edge preview channels, which contain the WebView2 prerelease bits. The [Edge preview channels](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-channels) consist of Canary (daily), Dev (weekly), and Beta (every 4 weeks).
+
+#### Why is prerelease testing important?
+Evergreen WebView2 is based on the evergreen Chromium platform that receives monthly major updates. Testing against prerelease channels is the de facto way for web developers to validate their apps against incoming browser updates for the past decade and more. A common discovery from investigations that we have conducted in the past year, is that we regression tends to be application specific. WebView2 is tested in a variety of scenarios, but it is impossible to cover all the scenarios that an app might encounter. This means that the only way to catch certain app specific regressions before it ships to stable is to test against prerelease channels.
+
+Some examples of the issues that could have been caught prior to stable prerelease testing include:
```

Comment:
garbled: "prior to stable prerelease testing".  stable is the opposite of prerelease, so, a contradiction in terms.
```suggestion
Some examples of the issues that could have been caught prior to Edge Stable by doing prerelease testing (such as in Edge Canary) include:
```

Reply:
Changed to the more neutral, positive, ambiguous wording:

Many issues in specific WebView2 apps can be discovered before WebView2 changes reach Edge Stable (WebView2 Runtime), by doing prerelease testing, such as in Edge Canary.  For example, the following issues have been caught:

@victorhuangwq - avoiding possibly implying that mistakes were made, when linking to specific issues for specific apps.  "stable prerelease testing" is self-contradictory.